src/amqp/consumers/consumer.ts

```diff
@@ -0,0 +1,34 @@
+import { Container } from '../../container';
+import { ConsumeMessage, Channel, Message } from 'amqplib';
+import { logger } from '../../logger';
+
+export abstract class Consumer {
+  queue: string;
```

Comment:
esse atributo é utilizado alem do construtor da classe?

Reply:
Alterado para somente leitura, ele precisa ser público para que as classes vHosts possam realizar o início do consumo das filas